### ISSUE TITLE
Setup connection using local Pack not working

### ISSUE THREAD

**Issue by FerdinandEn** [ARM-software/vscode-device-manager#11]
Type: Bug Report

**Describe the bug**
- Win 11
- VSCode 1.90.0
- CMSIS Solution 1.35.0
- Device Manager 1.12.1
- Environment Manager 1.1.17
- Keil Studio 1.18.2
- Target Device: STM32H753ZI

I have added the experimental pack STM32H7xx_DFP::4.0.0-dev0 as a local repository with `cpackget add` . I use a Nucleo-H753ZI board as hardware (STM32H753ZI). I can perform all development steps in Keil Studio as well as in uVison. However, the pack STM32H7xx_DFP::4.0.0-dev0 is not listed in the Device Manager, if I setup the connection. You can only see and use the normal pack STM32H7xx_DFP::3.3.1. 

My workaround is to use the package STM32H7xx_DFP::3.3.1 for debugging in Keil Studio.

**Comment by thegecko** [comment]
Device manager is not aware of custom packs. To use one please point at the full path to the `.pack` file in your `tasks.json` or `launch.json` configuration:

```json
{
    ...
    "cmsisPack": "<full path to your pack.pack file>"
    ...
}
```

**Comment by FerdinandEn** [comment]
The packs from

https://github.com/Open-CMSIS-Pack/pack-examples

are not available as packs, but only as  local packs. I can not build a pack with valid license.

I tried to enter the path to the pdsc file under CMSISPack. This did not work.

````
"cmsisPack": "${command:device-manager.getDevicePack}",
// "cmsisPack": "D:\\uni\\VScodeTest\\tttt\\pack-examples\\STM32H7xx_DFP.3.3.1.pack",
// "cmsisPack": "D:\\uni\\VScodeTest\\tttt\\pack-examples\\STM32H7xx_DFP\\Keil.STM32H7xx_DFP.pdsc",
"deviceName": "${command:cmsis-csolution.getDeviceNa
````

Using * pack I got the message:
ERROR(CLI223): No CMSIS-Packs found, CMSIS-Packs must be installed using the CMSIS-Pack Manager.

Using psdc I got the message

PDSC file not found

I can work with my workaround for my experiments. I just wanted to point out the problem.

--- 
Only for information

Here are my setting dialogs for the device

![DeviceManager](https://github.com/ARM-software/vscode-device-manager/assets/21216218/96fbb461-b371-438f-8913-7091a2e88fac)

Final setting

![DeviceManagerFinal](https://github.com/ARM-software/vscode-device-manager/assets/21216218/ea4fd1ab-291a-4aaa-9faf-702e8e83f8ed)

Despite the missing setting for RAM and Flash, I can now flash and debug.
